### ISSUE TITLE
debug doesn't work in OneDrive

### ISSUE THREAD

**Issue by ElectroLund** [cweijan/autohotkey-plus#65]
This is a curious issue.  I discovered that breakpoints aren't being obeyed on source files that are added to a OneDrive folder in Explorer.  This took a while to discover the culprit!

If I simply drag that file out of the folder and into a non-OneDrive folder... magic.  Breakpoints break as they should.

**Comment by biorpg** [comment]
Would the source file in question happen to have a total path length that is greater than 250 characters?

**Comment by ElectroLund** [comment]
Ahh, good idea.  Path length has bitten me before.  I just fired a project
up from inside my OneDrive path, verified the breakpoint doesn't work, and
the path looks like this:

C:\Users\nnn.nnnn\OneDrive - Company Name
Here\Documents\AutoHotkey\calendar picker.ahk

This one was only 94 characters.

**Comment by mark-wiemer** [comment]
Good to know this one is still active. OneDrive in general has been very weird for me with its behind-the-scenes file manipulation, that may be contributing here. We'll keep this one on the backlog, thanks for the repro steps.

**Comment by mark-wiemer** [comment]
@ElectroLund , have you considered using [vscode-autohotkey-debug](https://marketplace.visualstudio.com/items?itemName=zero-plusplus.vscode-autohotkey-debug)? It's a more advanced and well-maintained debugger

**Comment by ElectroLund** [comment]
Indeed, good suggestion.  That debugger has two immediate advantages:'
1) debugs just fine on AHK files inside of OneDrive paths.
2) also supports conditional breakpoints

Thanks!
